fms/modules/feedforward.py

```diff
@@ -470,13 +473,42 @@ class ConditionalFeedForward(nn.Module):
         The intermediate size for the expert networks.
     """
 
-    def __init__(self, num_experts: int, dim: int, intermediate_size: int):
+    def __init__(
+        self,
+        num_experts: int,
+        dim: int,
+        intermediate_size: int,
+        use_bias: bool = False,
+    ):
         super().__init__()
         self.num_experts = num_experts
         self.dim = dim
         self.intermediate_size = intermediate_size
         self.w13 = nn.Parameter(torch.empty(num_experts, 2 * intermediate_size, dim))
         self.w2 = nn.Parameter(torch.empty(num_experts, dim, intermediate_size))
+        self.use_bias = use_bias
+        self.w13_bias = torch.nn.Parameter(
```

Comment:
IIUC, the new `w13_bias` and `w2_bias` are only used if `self.use_bias = True`, so can we make code branch here and set them to `None` when `self.use_bias = False`?

Otherwise we seem to be adding weights that are never used by the model.